playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -17,6 +17,17 @@
   tags:
     - install
 
+- name: Install client packages
+  package:
+    name: "{{ suse_osh_client_packages }}"
+    state: present
+  register: install_client_packages
+  until: install_client_packages is success
+  retries: 5
```

Comment:
why retry and delay? Do we expect that zypper might be used in parallel?

Reply:
Again, copy & paste. But I assume that might have been the original concern, some zypper process already running....

Reply:
You can blame me for having this kind of things added in ansible-lint.
Recent versions of ansible-lint removed this, because it was considered "too nitpicky". Anyway, this is useful to ensure no network issues arises during installation of packages. As it is a network operation, retries is generally a good thing.

Reply:
Is that something that can be done globally maybe? So all `package` plugins calls do the retry/delay thing?

Reply:
AFAIU that would require changes to the package module in ansible. Or maybe some form for wrapper task for installing packages (used via `include_task` ?).
I think, if we'd want to go down that route we should add it to the backlog handle it in a separate PR.